package.json

```diff
@@ -34,7 +34,7 @@
     "taskcluster-client": "0.23.4",
     "taskcluster-lib-scopes": "^0.8.8",
     "taskcluster-lib-stats": "^0.8.7",
-    "taskcluster-lib-testing": "^0.10.1",
+    "taskcluster-lib-validate": "^0.4.2",
```

Comment:
nope...
But we get validate passed in now... So we don't need it as a dependency, right?